rpcserverhandlers_test.go

```diff
@@ -2284,6 +2315,229 @@ func TestHandleGetBlockchainInfo(t *testing.T) {
 	}})
 }
 
+func TestHandleGetBlock(t *testing.T) {
+	// Define variables related to block432100 to be used throughout the
+	// handleGetBlock tests.
+	blkHeader := block432100.Header
+	blk := dcrutil.NewBlock(&block432100)
+	blkHash := blk.Hash()
+	blkHashString := blkHash.String()
+	blkBytes, err := blk.Bytes()
+	if err != nil {
+		t.Errorf("error serializing block: %+v", err)
```

Comment:
Should be `Fatalf` here since otherwise it'll keep going and fail in strange ways.

Reply:
Good point - updated.